contracts/modules/common/BaseModule.sol

```diff
@@ -83,4 +83,30 @@ contract BaseModule is Module {
     function isOwner(BaseWallet _wallet, address _addr) internal view returns (bool) {
         return _wallet.owner() == _addr;
     }
+
+    /**
+     * @dev Helper method to invoke a wallet.
+     * @param _wallet The target wallet.
+     * @param _to The target address for the transaction.
+     * @param _value The value of the transaction.
+     * @param _data The data of the transaction.
+     */
+    function invokeWallet(address _wallet, address _to, uint256 _value, bytes memory _data) internal returns (bytes memory _res) {
+        bool success;
+        // solium-disable-next-line security/no-call-value
+        (success, _res) = _wallet.call.value(0)(
```

Comment:
Why do we explicitely set the value to zero?

Reply:
No good reason. Removed.